app/src/main/res/values/strings.xml

```diff
@@ -1,209 +1,304 @@
 <resources>
     <!--General Strings-->
-    <string name="app_name">SUSI AI</string>
-    <string name="send_msg_hint">Ask SUSI something…</string>
-    <string name="action_settings">Settings</string>
     <string name="action_about_us">About Us</string>
-    <string name="action_wall_settings">Wallpaper</string>
-    <string name="action_share">Share</string>
     <string name="action_log_out">Log out</string>
     <string name="action_login">Login</string>
-    <string name="settings_title">Chat Settings</string>
-    <string name="server_settings_title">Server Settings</string>
-    <string name="server_pref">Select Server</string>
-    <string name="server_select_summary">Select backend server for the app.</string>
-    <string name="search">Search</string>
-    <string name="select_background">Select background</string>
+    <string name="action_settings">Settings</string>
+    <string name="action_share">Share</string>
+    <string name="action_wall_settings">Wallpaper</string>
+
+    <string name="app_name">SUSI AI</string>
+
     <string name="background_no_wall">no_wall</string>
+
+    <string name="content">content</string>
+
+    <string name="dialog_retrieve_messages_title">Retrieving old messages…</string>
+
+    <string name="email_invalid">The e-mail address you entered doesn\'t belong to an account.
```

Comment:
Remove extra spaces.

Reply:
@arundhati24 I am talking about the space between `to an account.` and `Please check...`

Reply:
@akshatnitd Fixed that.